flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendMigrationTestBase.java

```diff
@@ -274,9 +277,10 @@ public void testKeyedListStateRegistrationFailsIfNewStateSerializerIsIncompatibl
 
             fail("should have failed");
         } catch (Exception expected) {
-            Assert.assertTrue(
-                    ExceptionUtils.findThrowable(expected, StateMigrationException.class)
-                            .isPresent());
+            assertThat(
+                            ExceptionUtils.findThrowable(expected, StateMigrationException.class)
+                                    .isPresent())
+                    .isTrue();
```

Comment:
i guess it could be replaced with usage of `assertThatThrownBy`
something like
```java
        assertThatThrownBy(
                        () ->
                                testKeyedListStateUpgrade(
                                        new ListStateDescriptor<>(
                                                stateName, new TestType.V1TestTypeSerializer()),
                                        new ListStateDescriptor<>(
                                                stateName,
                                                new TestType.IncompatibleTestTypeSerializer())))
                .isInstanceOf(StateMigrationException.class);
```